basic/src/test/java/org/jboss/hal/testsuite/test/configuration/singleton/SingletonPolicyTestCase.java

```diff
@@ -168,6 +174,9 @@ public void editCache() throws Exception {
                     .editAndSave(ConfigChecker.InputType.TEXT, CACHE, CACHE_RESOURCE)
                     .verifyFormSaved()
                     .verifyAttribute(CACHE, CACHE_RESOURCE);
+        } catch (NoSuchElementException e) {
+            //TODO remove this when issue is resolved
```

Comment:
Please remove the TODO comment. We don't want to remove regression assertion messages.